webpack/components/extensions/HostDetails/HostPackages/HostPackagesConstants.js

```diff
@@ -1,4 +1,5 @@
 export const HOST_PACKAGES_KEY = 'HOST_PACKAGES';
+export const HOST_PACKAGES_INSTALL_KEY = 'HOST_PACKAGES_KATELLO_AGENT_INSTALL';
```

Comment:
How do you find out the value for the key? Where is it got set?

Reply:
It's set right here!  The API middleware then puts the response in the Redux store under that key. More info here: https://github.com/theforeman/foreman/blob/develop/webpack/stories/docs/api-middleware.stories.mdx